--- /home/a1648400/plc-2.0/Makefile	2015-04-28 02:51:27.000000000 +0930
+++ /home/a1648400/plc-2.0_new/Makefile	2016-12-06 22:32:23.000000000 +1030
@@ -19,14 +19,14 @@
 
 # set the path of the cfitsio lib. 
 # bewared that cfitsio must have been compiled with the option "make shared"
-CFITSIOPATH := /usr/local
+CFITSIOPATH := $(HOME)/cfitsio
 #CFITSIOPATH := /softs/cfitsio/3.24
 # you have a CFITSIO lib in a weird location, also set those
 CFITSIO_INCPATH := $(CFITSIOPATH)/include
 CFITSIO_LIBPATH := $(CFITSIOPATH)/lib
 
 #define your compilers and stuff
-CC = gcc
+CC = icc
 FC = ifort
 
 # ifort
@@ -36,8 +36,8 @@
 # (see https://software.intel.com/en-us/articles/gcc-49-openmp-code-cannot-be-linked-with-intel-openmp-runtime)
 
 # on my mac I got
-IFORTLIBPATH = /usr/bin/ifort-2011-base/compiler/lib
-IFORTRUNTIME = -L$(IFORTLIBPATH) -lintlc -limf -lsvml -liomp5 -lifportmt -lifcoremt -lpthread
+# IFORTLIBPATH = /usr/bin/ifort-2011-base/compiler/lib
+# IFORTRUNTIME = -L$(IFORTLIBPATH) -lintlc -limf -lsvml -liomp5 -lifportmt -lifcoremt -lpthread
 
 # on a linux machine, ifort 11.1
 #IFORTLIBPATH = /softs/intel/fce/11.1.075/lib/intel64
@@ -47,6 +47,9 @@
 #IFORTLIBPATH = /softs/intel/xe/composer_xe_2013_sp1.1.106/compiler/lib/intel64
 #IFORTRUNTIME = -L$(IFORTLIBPATH) -lintlc -limf -lsvml -liomp5 -lifport -lifcoremt -lirc -lpthread
 
+IFORTLIBPATH = /apps/software/ifort/2015.3.187/lib/intel64
+IFORTRUNTIME = -L$(IFORTLIBPATH) -lintlc -limf -lsvml -liomp5 -lifport -lifcoremt -lpthread
+
 # gfortran
 # if you are using gfortran set here where the lib are installed
 # and check the runtime libs
@@ -54,8 +57,8 @@
 GFORTRANRUNTIME = -L$(GFORTRANLIBPATH) -lgfortran -lgomp
 
 # if you are on linux and using mkl, you need to set this 
-MKLROOT = 
-LAPACKLIBPATHMKL = 
+MKLROOT = /apps/software/ifort/2015.3.187/mkl
+LAPACKLIBPATHMKL = $(MKLROOT)/lib/intel64
 #some example
 #MKLROOT = /softs/intel/mkl/10.2.6.038/
 # on mkl 10.3
@@ -72,13 +75,13 @@
 COLORS = 1
 
 #set echo to echo -e to have colourized output on some shell
-ECHO = echo 
+ECHO = echo -e
 
 
 # what is the openmp option for your C compiler (leave empty to cmpile without openmp)
-COPENMP = -fopenmp
+COPENMP = -openmp -Ofast
 # what is the openmp option for your F90 compiler (leave empty to cmpile without openmp)
-FOPENMP = -openmp
+FOPENMP = -openmp -Ofast
 
 # what is the 32/64 bit option for your C compiler (leave empty if you don't want to know)
 CM64 =
